index.html

```diff
@@ -0,0 +1,338 @@
+
+            <a href="#" class="icon" >
+              <i class="fa fa-bars"></i>
+            </a>
+          </nav>
+        
+        </header>
+  
+
+        <section>
+          <div class="title-main ">
+            <h1 class="name-text">I’m Benjamin Agyekum Glad<br>to see you!
+            </h1>
+          </div>
+          <div class="sub-text ">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or websites Look through some of my work and experience! if you
+              like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </p>
+          </div>
+          <p class="mini-text ">LET’S CONNECT</p>
+          <div class="social-media ">
+            <ul class="media-icon ">
+              <li><a href="#" class="fa fa-twitter"></a></li>
+
+              <li><a href="#" class="fa fa-facebook-f"></a></li>
+
+              <li><a href="#" class="fa fa-linkedin"></a></li>
+
+              <li><a href="#" class="fa fa-github"></a></li>
+
+              <li><a href="#" class="fa fa-angellist"></a></li>
+            </ul>
+          </div>
+        </section>
+      </div>
+
+      
+<section>
+    <div class="grid-wrapper">
+        <div class="work-card layout">
+            <div class="Work-card-head ">
+                <div class="Work-card-body">
+                    <div class="Work-card-img ">
+                    </div>
+                    <h3 class="work-card-title ">Tonic</h3>
+                    <div class="work-card-dot ">
+                        <div class="card-text-main">CANOPY</div>
+                        <div class="work-card-dot-s"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">Back End Dev</div>
+                        <div class="work-card-dot-em2"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">2015</div>
+                    </div>
+                    <p class="card-subtext ">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
```

Comment:
- [ ] As said by the previous reviewer kindly make the button's interaction active, hover, and pressed as the Figma design

Reply:
the button has hover on it so i dont know what you mean here

Reply:
its on the class button1